packages/reports/addon/components/navi-action-list.hbs

```diff
@@ -1,103 +1,108 @@
 {{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-<ul class="navi-action-list actions" ...attributes>
-
+<div class="navi-actions flex-1" ...attributes>
   {{!-- Clone only enabled on a saved report --}}
-  <li class="action">
-    {{#if @item.isNew}}
-      <LinkTo
-        class="navi-reports-index__report-control clone navi-report__action-link--force-disabled"
-        disabled="true"
-        @route="reports.report.clone"
-        @model={{@item.id}}
-      >
-        <NaviIcon @icon="copy" />
-        <EmberTooltip @popperContainer="body" @text="Save report to enable clone" />
-      </LinkTo>
-    {{else}}
-      <LinkTo @route="reports.report.clone" @model={{@item.id}} class="navi-reports-index__report-control clone">
-        <NaviIcon @icon="copy" />
-        <EmberTooltip @popperContainer="body" @text="Clone the report" />
-      </LinkTo>
-    {{/if}}
-  </li>
+  {{#if @item.isNew}}
+    <LinkTo
+      class="navi-reports-index__report-control clone navi-report__action-link--force-disabled"
+      disabled="true"
+      @route="reports.report.clone"
+      @model={{@item.id}}
+    >
+      <DenaliIcon disabled="true" @size="small" @icon="copy-file" class="p-4 link" />
+      <EmberTooltip @popperContainer="body" @text="Save report to enable clone" />
+    </LinkTo>
+  {{else}}
+    <LinkTo @route="reports.report.clone" @model={{@item.id}} class="navi-reports-index__report-control clone">
+      <DenaliIcon class="clone p-4 link" @size="small" @icon="copy-file" />
+      <EmberTooltip @popperContainer="body" @text="Clone the report" />
+    </LinkTo>
+  {{/if}}
 
-  <li class="action">
   {{!-- Export only enabled on a validated report --}}
-    {{#let (feature-flag "exportFileTypes") as |exportFileTypes|}}
-      {{#let (component (concat "report-actions/" (if (gt exportFileTypes.length 1) "multiple-format-export" "export"))) as |ExportAction|}}
-        <ExportAction
+  {{#let (feature-flag "exportFileTypes") as |exportFileTypes|}}
+    {{#let (component (concat "report-actions/" (if (gt exportFileTypes.length 1) "multiple-format-export" "export"))) as |ExportAction|}}
```

Comment:
Can we wrap this block in `{{#if (gt exportFileTypes.length 0)}}` so that export only shows up when expected